pkg/leech/ingest_logs.go

```diff
@@ -124,10 +124,12 @@ func (f *IngestLogsFn) StartBundle(ctx context.Context) error {
 		return fmt.Errorf("failed to read private key: %w", err)
 	}
 	ghAppConfig := githubapp.NewConfig(f.GitHubAppID, f.GitHubInstallID, pk)
+	// Why not f.ghApp = githubapp.New(ghAppConfig)?
```

Comment:
Initial implementation didn't assign it to f.ghApp and when that change was made it wasn't cleaned up.